ecs-spot-terraform/README.MD

```diff
@@ -0,0 +1,38 @@
+# Create ECS cluster with Spot and On-demand capacity providers using Terraform 
+
+Use this Terraform template to create an ECS cluster with two capacity providers connected to two Auto Scaling groups. First one with 100% EC2 Spot Instances capacity, configured with Spot best practices to get optimal capacity. Second group is 100% On-demand capacity to support workloads that are not fault tolerant.
+
+This ECS cluster will have a default capacity providers strategy to balance tasks between on-demand and Spot instances. Which can be adjusted to fit the capacity needs for different workloads.
```

Comment:
It may be obvious, but can you add "it will balance tasks evenly between On-Demand and Spot instances"? Add also a link to Capacity Provider Strategy docs.